src/content/releases/release-v0.2.0.mdx

```diff
@@ -0,0 +1,119 @@
+version: "0.2"
+date: 2024-10-04
+description: "TODO"
+published: true
+---
+
+Operately's latest release is packed with improvements designed to make your
+company’s operations smoother and more efficient. Here’s what’s new!
+
+## Project Retrospectives that drive results
+
+We've upgraded our project retrospectives. Now you can squeeze more value from
+every project you complete. Our new retrospectives let you easily review past
+projects, discuss improvements openly, and learn from your team's insights.
+
+We built this feature because we know how hard it is to pause and reflect when 
+you're moving fast. But those moments of reflection are gold. They help you avoid
+repeating mistakes and double down on what's working.
+
+Try it out. Open a recent project and start a retrospective. You might be 
+surprised by the insights you uncover.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/envGi8G.png" alt="Project Retrospective" className="rounded-lg border border-gray-300" />
+</div>
+
+## Onboard and Offboard people faster
+
+We've streamlined team management for company admins. The new admin page puts
+more control at your fingertips.
+
+Company admins now have more control over managing and updating profiles.
+Whether it's assigning manager-reporter relationships or changing someone's
+title, it's all doable from the new admin page. Need to remove someone who has
+left the company? Their account will now be automatically suspended,
+streamlining off-boarding without the hassle.
+
+As an admin click on the company dropdown -> Company Admin -> Manage Team Members.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/ljimvRg.png" alt="People Management" className="rounded-lg border border-gray-300" />
+</div>
+
+## Manage who does what on your projects
+
+We've upgraded how Operately handles project team changes. Now it's easier to
+match people's roles to their real responsibilities.
+
+Say your lead developer Sarah steps up to manage a critical feature launch. In
+seconds, you can promote her from contributor to project champion. She gets the
+right access, and you keep moving. Or three software engineers join the project, 
+each responsible for a different part of the system. Now, you add all three at once, 
+assigning them specific responsibilities and keeping everyone in the loop.
+
+Why does this matter? Because in the real world, roles change. People step up.
+New folks join. You shouldn't need an IT degree to keep your project moving when 
+this happens. We built this because we've felt this pain ourselves. Unclear 
+responsibilities kill momentum. They breed confusion.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/HX85OsP.png" alt="People Management" className="rounded-lg border border-gray-300" />
```

Comment:
By the title of the segment I would expect to see detailed, project specific responsibilities on the project. Having it say "Software Developer" directly contradicting what the point of this is.